src/library/commissioner_impl.hpp

```diff
@@ -251,6 +251,17 @@ class CommissionerImpl : public Commissioner
 
     void HandleJoinerSessionTimer(Timer &aTimer);
 
+    bool IsDiagQueryId(uint16_t aQueryId) const
```

Comment:
Can this change be merged into a PR which actually use it? We don't need to split PRs small like this. For example, it's actually harder to review this PR without knowing how this method will be used.

Reply:
make sense, Let me cancel it and merge it to #275. Thanks.